### PR TITLE
Package Dependency Ansible Tooling: Updated dependency tooling logic updates

### PR DESCRIPTION
**Pull request by jonesdt**

### CODE REVIEW

File: pkg_list.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python
+
+import os
+import sys
+
+
+#path to dependent packages
+dependent_pkgs_path = '/srv/repos/dependencies/rhel7/dependencies'
```
Comment (jjalvare) [inline pkg_list.py]: constant settings will break in the future if future  changes move the path. you should look at where you can get this path programmatically and also update it with the changes from the user

Comment (jonesdt) [reply pkg_list.py]: I agree. Not sure as to why this was incorporated into this pull request. Ansible playbook logic does not reference pkg_list.py. If needed I will resubmit excluding this update or once merged, we can remove from master.

File: software/wmla120_ansible/engr_mode_env_pre_install_gather.yml [hunk 2]
```diff
@@ -86,39 +87,40 @@
 
 - name: Check if dlipy2 pre data exists (PIP)
   stat:
-    path: "~/{{ dlipy2_pip }}"
+    path: "{{ ansible_env.HOME }}/{{dlipy2_pip}}"
+>>>>>>> 4a16ee2258c6900f005fbf3b751a4a304a739bb3
```
Comment (jjalvare) [inline software/wmla120_ansible/engr_mode_env_pre_install_gather.yml]: hmm ... have you unit tested this code after git merge? Seems that this is incorrect.

Comment (jonesdt) [reply software/wmla120_ansible/engr_mode_env_pre_install_gather.yml]: Yes..just needed cleanup due to resolved conflicts.

**Comment by jonesdt** [comment]
Ran TOX and completed successfully with no errors.

**Pull request merged**